OSPSuite.Presentation.Importer/Core/DataFormat/ColumnsDataFormat.cs

```diff
@@ -94,7 +94,7 @@ private void extractGeneralParameters(List<string> keys, IUnformattedData data)
 
       public IList<Dictionary<Column, IList<double>>> Parse(IUnformattedData data)
       {
-         var groupByParams = Parameters.Where(p => p.Configuration.Type == ParameterConfiguration.DataFormatParameterType.GroupBy).Select(p => (p.ColumnName, data.Headers[p.ColumnName].ExistingValues));
+         var groupByParams = Parameters.Where(p => p.Type == ParameterConfiguration.DataFormatParameterType.GroupBy).Select(p => (p.ColumnName, data.Headers[p.ColumnName].ExistingValues));
```

Comment:
This is a sign of a missing abstraction in my opinion when the same code is seen over and over again. 
I have defined the Type in the IDataFormatParmaeter. Configuration seems to be more of an internal stuff than anything else